error.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div>
+    <qiskit-ui-shell variant="hide-account" @on-click="onClick" />
+    <div class="error-page">
+      <div class="cds--grid">
+        <div class="cds--row">
+          <div class="cds--col-lg-4">
+            <h1 class="error-page__title">{{ props.error?.statusCode }}</h1>
+            <p>
+              {{ props.error?.message }}
+            </p>
+            <p class="error-page__message">
+              Or as we like to say,
+              <!-- eslint-disable vue/no-v-html -->
+              <span v-html="randomMessage" />
+              <!-- eslint-enable -->
```

Comment:
We can use "eslint-disable-next-line" instead of disabling and enabling again:

```suggestion
              <!-- eslint-disable-next-line vue/no-v-html -->
              Or as we like to say, <span v-html="randomMessage" />
```